### DISCUSSION TITLE
Vite package

### DISCUSSION THREAD

**Discussion by kadirgun** [goravel/goravel#502] [Q&A]
Is there a package where we can integrate Vite dev and build environment? Like the `@vite` directive in Laravel.

```html
<head>
{{ vite resources/css/app.css resources/js/app.js }}
</head>
```

**Comment by hwbrzzl** [comment]
Emm, I have no idea about this, you can try to find such a template package, then use it in Goravel, the view feature of Goravel is very easy to extend.

**Reply by kadirgun** [reply]
I tried to develop a package myself but I couldn't figure out how to pass a function into a view. I think template functions are defined with `gin.NewTemplate()` when the application boots.

**Comment by hwbrzzl** [comment]
You can use your own template package here:

<img width="905" alt="image" src="https://github.com/user-attachments/assets/1a9f2980-8884-48e5-9e42-1912449f6953">
